### ISSUE TITLE
Add Smoothing Period in ADX

### ISSUE THREAD

**Issue by myalgomate** [DaveSkender/Stock.Indicators#1019]
In Continuation of the Discussion related to ADX.

https://github.com/DaveSkender/Stock.Indicators/discussions/1015#discussioncomment-4887885

**Comment by myalgomate** [comment]
Hi Dave
Hope you are doing well.
I just wanted to know if you could help me with the above issue.

Thanks in advance.

Regards,
Harsh

**Comment by DaveSkender** [comment]
> Hi Dave Hope you are doing well. I just wanted to know if you could help me with the above issue.

Yes.  I've put it in the backlog, so it is something I intend to implement, but not immediately
Given other more widely needed features, I do not expect to work on this anytime soon.
If you need it sooner, please consider [contributing the code](https://dotnet.stockindicators.dev/contributing/#content) in a Pull Request.

If you decide to take this one on, please let me know so I can reserve it for you.

**Comment by myalgomate** [comment]
Hi Dave,

Sorry, but I didn't understand what you were saying.?
It would be really great if you can add smoothing parameters as arguments in ADX.

Thanks

**Comment by DaveSkender** [comment]
I cannot work on this now.

**Comment by myalgomate** [comment]
okay, Thank you.
Please do at your convenience.

**Comment by kresch23** [comment]
@DaveSkender I also need the Smoothing parameter in the ADX function. Can I possibly support that we can implement this?

**Comment by DaveSkender** [comment]
> Can I possibly support that we can implement this?

If you want to create an overload method here and submit a pull request, I’d be able to review it.  Contributions are greatly appreciated.

**Comment by kresch23** [comment]
@DaveSkender I have looked at the code and am apparently already confused about the structure. My knowledge is probably not sufficient :/ I would be willing to pay for you to implement this, would that be an option?

**Comment by DaveSkender** [comment]
> I would be willing to pay for you to implement this, would that be an option?

Yes.  Many of [our sponsorship tiers](https://github.com/sponsors/facioquo?frequency=one-time) come with prioritization of work.
